internal/provider/resource_tsuru_plan.go

```diff
@@ -117,7 +118,6 @@ func resourceTsuruPlanRead(ctx context.Context, d *schema.ResourceData, meta int
 		return nil
 	}
 
-	d.SetId("")
```

Comment:
@juliolustosa could you revert this change ?

Reply:
ok